recog/src/main/java/com/rapid7/recog/provider/RecogMatchersProvider.java

```diff
@@ -113,8 +113,12 @@ private void parseFromWalkablePath(Path path) {
           try (Reader reader = Files.newBufferedReader(file)) {
             int extIndex = fileName.lastIndexOf(".xml");
             RecogMatchers matchers = parser.parse(reader, extIndex > 0 ? fileName.substring(0, extIndex) : fileName);
-            matchersByFileName.put(fileName, matchers);
-            matchersByKey.put(matchers.getKey(), matchers);
+            if (null != matchers) {
```

Comment:
Is there a reason for putting the `null` on the left hand side? Concerns around swapping the side to maintain consistency within the project?

Reply:
No problem here, habit on our side